### DISCUSSION TITLE
heading font color

### DISCUSSION THREAD

**Discussion by JesseRacine** [zim-desktop-wiki/zim-desktop-wiki#2422] [Q&A] [answered]
I'm in Windows and I wanted to know if there is a way to change the font color for the headings?

**Comment by k4nzdroid** [comment, marked as answer]
See https://zim-wiki.org/manual/Help/Config_Files.html

Edit style.conf and add the following configuration:
```
[Tag h1]
foreground=#4e9a06

[Tag h2]
family=
foreground=white
background=green
```

**Reply by introt** [reply]
There's also the Text View plugin: https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/1802#discussioncomment-1798274
